client/src/App.vue

```diff
@@ -1996,6 +2003,10 @@
 			<span>
 				<a href="mailto:mtgadraft@gmail.com">Contact</a>
 			</span>
+			<span>
+				<a href="https://discord.gg/XscXXNw"><i class="fab fa-discord"></i></a>
+				<a href="https://github.com/Senryoku/MTGADraft"><i class="fab fa-github"></i></a>
+			</span>
 			<span>
 				Get
 				<a href="https://magic.wizards.com/fr/mtgarena" target="_blank" rel="noopener nofollow">Magic: The Gathering Arena</a>
```

Comment:
Have some suggestions:

```suggestion
			<span>
				<a href="mailto:mtgadraft@gmail.com"><i class="fas fa-envelope"></i></a>
			</span>
			<span>
				<a href="https://discord.gg/XscXXNw" target="_blank" rel="noopener nofollow"><i class="fab fa-discord"></i></a>
				<a href="https://github.com/Senryoku/MTGADraft target="_blank" rel="noopener nofollow""><i class="fab fa-github"></i></a>
			</span>
			<span>
				Get
				<a href="https://magic.wizards.com/en/mtgarena" target="_blank" rel="noopener nofollow">Magic: The Gathering Arena</a>
```